### DISCUSSION TITLE
Created Installation Video of Deep Live Cam for Windows

### DISCUSSION THREAD

**Discussion by fahdmirza** [hacksider/Deep-Live-Cam#110] [Show and tell]
Hi Team,
I hope you don't mind me sharing as I am very excited to share this step-by-step installation video of your awesome tool :

https://youtu.be/MICScLVM1x8?si=gRMEyB26CYrKat9r

Thanks.

Fahd

**Comment by PBandJ-Dev** [comment]
Cheers mate I followed your instructions and got it working for Video but not live (Ive raised a request for it so dont worry), but thanks you made a great video and it was easy to follow, keep up the good work.

**Reply by fahdmirza** [reply]
Good to know, thanks

**Comment by cevaliber** [comment]
How to setup the Deep Live Cam to go live, any videos? or instructions ?

**Comment by Winstonjeff** [comment]
pls can you help me for the installation?

**Comment by TouchFotos** [comment]
Hello Fahdmirza, I understand your pc has NVIDIA. Maybe that's why it was easy for you to install it. My pc run on Intel(Iris) XE Graphics. I'll attach a picture of my PC details, so you can assist me on what to do next. Please somebody respond

![2025-01-11 (1)](https://github.com/user-attachments/assets/9d1dfb04-bf0e-4429-809c-5c367a2e5545)


![2025-01-09 (1)](https://github.com/user-attachments/assets/95a2a6a9-f315-41bd-8ac4-51772313d991)

**Reply by 77-10** [reply]
no
